controller/Dockerfile

```diff
@@ -10,7 +10,8 @@ RUN curl -sSL -o /usr/local/bin/etcdctl https://s3-us-west-2.amazonaws.com/opdem
     && chmod +x /usr/local/bin/etcdctl
 
 # install confd
-RUN curl -sSL -o /usr/local/bin/confd https://github.com/kelseyhightower/confd/releases/download/v0.8.0/confd-0.8.0-linux-amd64 \
+ENV CONFD_VERSION 0.9.0
```

Comment:
I realize it's a bit of a micro-optimization, but that's how `docker pull` makes me think. I think the one-line change accomplishes your goal of easier editing as well, but I won't hold up the PR for this change.